src/app/reference/replay-cli/source-maps/page.md

```diff
@@ -8,89 +8,237 @@ Source maps exist to try and preserve the original debugging experience as best
 
 ## Upload Process
 
-For production use cases where sourcemaps may not be exposed publicly, we provide tooling for preemptively uploading sourcemaps to our servers so that when replays are made, they can make use of these maps. We provide both a CLI command, and a Webpack plugin to make integrating this into your build process as easy as possible.
+For production use cases where sourcemaps may not be exposed publicly, we provide tooling for preemptively uploading sourcemaps to our servers so that when replays are made, they can make use of these maps and you can still debug the recordings with the original source code. We provide both a CLI command and a Webpack plugin to make integrating this into your build process as easy as possible.
 
 ### API Keys
 
 We recommend using the `REPLAY_API_KEY` environment variable to set up the API for use in all of our following examples, but each also allows passing the key directly as an argument. API Keys are available in the [Team Settings](/reference/ci-workflows/generate-api-key) modal.
 
-### CLI
+**Be sure to add the `REPLAY_API_KEY` for the workspace to the build and deployment settings for your project**, such as in Vercel or Github Actions - it needs to be available during the build step in order for the upload logic to upload successfully.
 
-You can upload source maps using the `upload-sourcemaps` command in the Replay CLI.
+## CLI
+
+You can upload source maps using the `replayio upload-source-maps` command in the [Replay CLI](/reference/replay-cli/commands).
 
 The simplest usage will be running the following after your production JS build has finished.
 
 ```sh
-npx replayio upload-sourcemaps --group <Version or SHA> <buildOutputDir>
+npx replayio upload-source-maps --group <Version or SHA> <buildOutputDir>
 ```
 
 You must ensure that your build tool is configured to output sourcemaps into the build directory you provide.
 
-### Webpack
+### CLI Options
+
+The `replayio upload-source-maps` command has options available to help locate and filter which sourcemaps are available. Currently, those options are:
+
+```bash
+-g, --group <name>       The name to group this source map into, e.g. A commit SHA or release version.
+-x, --extensions <exts>  A comma-separated list of file extensions to process; default ".js,.map"
+-i, --ignore <pattern>   Ignore files that match this pattern
+--root <dirname>         The base directory to use when computing relative paths
+```
```

Comment:
We should probably give an example of where this is used as well or atleast show some pseudo code where there's a webpack build followed by this command